style.css

```diff
@@ -81,6 +81,81 @@ body {
   }
 }
 
+.btn-popup {
+  padding: 8px;
+  border-radius: 5px;
+  margin: 8px 0;
+  transition: all 0.5s;
+  border: 2px solid #7f8cff;
+  color: #6070ff;
+  font-size: 18px;
+  font-weight: 500;
+  background-color: #fff;
+}
+
+.popup {
+  padding: 20px;
+  width: 950px;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  right: 150px;
+  display: none;
+  font-family: "Poppins", sans-serif;
+  z-index: 99999;
+  text-align: center;
+  backdrop-filter: blur(4px);
+  background-color: #e5e5e5e6;
+  border-radius: 16px;
+  margin: 30px;
+}
+
+.newElement {
+  background-color: #fff;
+  width: 0%;
```

Comment:
- [ ] Please fix the popup card styling in the mobile view by centering the positioning of the card first and give it the correct width value and fix the positioning of the line break above the card buttons, and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/162213191-965a3df2-407b-415c-8893-687d2f0fe85c.png)  | ![image](https://user-images.githubusercontent.com/68796232/162213085-12b15869-47fe-41bd-bdf4-3c4203df4e5d.png)  |

Reply:
The mobile popup looks good now 👍🏽